### ISSUE TITLE
pretrained model's performance

### ISSUE THREAD

**Issue by adsf0427** [DiT-3D/DiT-3D#2]
The released pretrained model's performance is lower worse what is listed in the paper

**Comment by Wanghc233** [comment]
Bro， can i add your contact information and you communicate with the code to run the problem?

**Comment by DiT-3D** [comment]
Hi, thanks for your interest in our work. Would you mind sharing the performance you get? Also please check the visualization for the qualitative generation.

**Comment by LiXiaoYao-0** [comment]
Your work is very meaningful. However, when I used test.py to verify your chair model, the result was much lower than the result of your paper. I didn't make any changes. Is there any special setting that I didn't pay attention to? Can you help me? 

my run test.py:
python test.py --dataroot /opt/data/private/data/data/ShapeNetCore.v2.PC15k/ \
    --category chair \
    --model_type 'DiT-S/4' \
    --voxel_size 32 \
    --model checkpoints/checkpoint.pth

result:
'1-NN-CD-acc': 0.652567982673645,
 '1-NN-CD-acc_f': 0.8383685946464539,
 '1-NN-CD-acc_t': 0.4667673707008362,
 '1-NN-EMD-acc': 0.6080060601234436,
 '1-NN-EMD-acc_f': 0.7296072244644165,
 '1-NN-EMD-acc_t': 0.4864048361778259,
 'lgan_cov-CD': 0.3867069482803345,
 'lgan_cov-EMD': 0.4274924397468567,
 'lgan_mmd-CD': 0.0026874206960201263,
 'lgan_mmd-EMD': 0.01597818173468113,
 'lgan_mmd_smp-CD': 0.0020523727871477604,
 'lgan_mmd_smp-EMD': 0.013778760097920895

your paper results:
1NN-CD     49.11 
1NN-EMD  50.73 
COV-CD     52.45 
COV-EMD  54.32

**Comment by DiT-3D** [comment]
Hi, thanks for your response. Note that the pre-trained model is only used as a test case, and we reported the XL models to the main table in our paper for final comparisons.

**Comment by edward1997104** [comment]
I would like to ask about the performance too. It seems that I have obtained a worse performance (using test.py) as suggested above even for the S model:

{'lgan_mmd-CD': 0.0026381739880889654, 'lgan_cov-CD': 0.4063444137573242, 'lgan_mmd_smp-CD': 0.0021268699783831835, 'lgan_mmd-EMD': 0.015758590772747993, 'lgan_cov-EMD': 0.44712990522384644, 'lgan_mmd_smp-EMD': 0.015100261196494102, '1-NN-CD-acc_t': 0.43353474140167236, '1-NN-CD-acc_f': 0.80513596534729, '1-NN-CD-acc': 0.6193353533744812, '1-NN-EMD-acc_t': 0.5196374654769897, '1-NN-EMD-acc_f': 0.7069486379623413, '1-NN-EMD-acc': 0.6132930517196655}

Table in the main paper:
![image](https://github.com/DiT-3D/DiT-3D/assets/31784490/c01e4a77-2cc9-4cac-89c4-dccb94eda118)

It would be very nice to have some pointers for knowing if I have missed in the evaluation. Thank you so much!

**Comment by DiT-3D** [comment]
Hi, thanks for your interest in our work! Please note that we calculated the score for the validation set and averaged the metrics across all batches. We tested the weights using 4 A100-40G GPUs with a batch size of 16.

**Comment by JudgeLJX** [comment]
> Your work is very meaningful. However, when I used test.py to verify your chair model, the result was much lower than the result of your paper. I didn't make any changes. Is there any special setting that I didn't pay attention to? Can you help me?
> 
> my run test.py: python test.py --dataroot /opt/data/private/data/data/ShapeNetCore.v2.PC15k/ --category chair --model_type 'DiT-S/4' --voxel_size 32 --model checkpoints/checkpoint.pth
> 
> result: '1-NN-CD-acc': 0.652567982673645, '1-NN-CD-acc_f': 0.8383685946464539, '1-NN-CD-acc_t': 0.4667673707008362, '1-NN-EMD-acc': 0.6080060601234436, '1-NN-EMD-acc_f': 0.7296072244644165, '1-NN-EMD-acc_t': 0.4864048361778259, 'lgan_cov-CD': 0.3867069482803345, 'lgan_cov-EMD': 0.4274924397468567, 'lgan_mmd-CD': 0.0026874206960201263, 'lgan_mmd-EMD': 0.01597818173468113, 'lgan_mmd_smp-CD': 0.0020523727871477604, 'lgan_mmd_smp-EMD': 0.013778760097920895
> 
> your paper results: 1NN-CD 49.11 1NN-EMD 50.73 COV-CD 52.45 COV-EMD 54.32

Hi, Did you reproduce their results?

**Comment by matteo-bastico** [comment]
> Hi, thanks for your interest in our work! Please note that we calculated the score for the validation set and averaged the metrics across all batches. We tested the weights using 4 A100-40G GPUs with a batch size of 16.

Hello, thank you for this accurate information. Is 16 the total batch size you used across all GPUs or the batch size per GPU?
